src/generator/models/declaration.ts

```diff
@@ -249,7 +249,7 @@ export function fieldTypeToGraphQLType(
           return StandardgraphQLScalarTypes.Float
         }
         case 'BigInt': {
-          return StandardgraphQLScalarTypes.String
```

Comment:
Just realizing this identifier as written before by me looks wrong, should have been `StandardGraphQLScalarTypes`.